universum/modules/output/html_output.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, *args, **kwargs):
         self.artifact_dir_ready = False
         self._log_buffer = []
         self._block_level = 0
+        self.current_dir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
In my opinion, "current_dir" does not entirely describe the meaning of this variable.
Probably, "module_dir" would be better name.